### PR TITLE
OHRM5X-2219: Add Include Employee Filter To Employee Claims

### PR DESCRIPTION
**Pull request by ManulMax**

### CODE REVIEW

**Comment by ManulMax** [comment]
**need to add/improve api testing

File: src/client/src/core/util/helper/filesize.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@
  */
 
 const BYTE = 1;
-const KILO_BYTE = 1024;
+const KILO_BYTE = 1000;
 const MEGA_BYTE = 1048576; // 1024 * 1024
 const GIGA_BYTE = 1073741824; // 1024 * 1024 * 1024
```
Comment (RajithaKumara) [inline src/client/src/core/util/helper/filesize.ts:22]:
```suggestion
const MEGA_BYTE = 1_000_000; // 1000 * 1000
const GIGA_BYTE = 1_000_000_000; // 1000 * 1000 * 1000
```

File: src/client/src/core/util/helper/__tests__/filesize.spec.ts
Comment (RajithaKumara) [inline src/client/src/core/util/helper/__tests__/filesize.spec.ts:1]: Add few test cases to cover MB and GB

File: src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php [hunk 2]
```diff
@@ -77,6 +77,7 @@ class EmployeeClaimRequestAPI extends Endpoint implements CrudEndpoint
     public const PARAMETER_TO_DATE = 'toDate';
     public const PARAMETER_MODEL = 'model';
     public const MODEL_DEFAULT = 'default';
+    public const FILTER_INCLUDE_EMPLOYEES = 'includeEmployees';
```
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php:80]:
```php
    public const MODEL_DEFAULT = 'default';
    public const MODEL_SUMMARY = 'summary';
    public const MODEL_MAP = [
        self::MODEL_DEFAULT => EmployeeClaimRequestModel::class,
        self::MODEL_SUMMARY => ClaimRequestSummaryModel::class,
    ];
    public const FILTER_INCLUDE_EMPLOYEES = 'includeEmployees';
```
move new param down

File: src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php [hunk 3]
```diff
@@ -291,6 +292,12 @@ protected function setClaimRequest(ClaimRequest $claimRequest, int $empNumber):
      *         @OA\Schema(type="DateTime")
      *     ),
      *     @OA\Parameter(
+     *         name="includeEmployees",
+     *         in="query",
+     *         required=false,
+     *         @OA\Schema(type="string")
```
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/Api/EmployeeClaimRequestAPI.php:298]:
```suggestion
     *         @OA\Schema(type="string", enum=ClaimRequestSearchFilterParams::INCLUDE_EMPLOYEES)
```
OR
```suggestion
     *         @OA\Schema(type="string", enum={"onlyCurrent", "onlyPast", "currentAndPast"})
```

Double check and correct https://github.com/search?q=repo%3Aorangehrm%2Forangehrm%20name%3D%22includeEmployees%22%2C&type=code these places also

File: src/plugins/orangehrmClaimPlugin/Dto/ClaimRequestSearchFilterParams.php [hunk 4]
```diff
@@ -67,6 +81,22 @@ public function __construct()
         $this->setSortOrder('DESC');
```
Comment (RajithaKumara) [inline src/plugins/orangehrmClaimPlugin/Dto/ClaimRequestSearchFilterParams.php:81]:
```suggestion
        $this->setSortOrder(ListSorter::DESCENDING);
```

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**